src/app/hooks/useViewTracker/index.tsx

```diff
@@ -168,28 +168,28 @@ const getComponentViewTracker = (eventTrackingData?: EventTrackingData) => {
     useReverb,
   ]);
 
-  return async (element: HTMLElement) => {
-    if (!element || !trackingIsEnabled || eventSent) {
-      return;
-    }
-    if (!observer.current) {
-      await initObserver();
-    }
+  const viewTracker = useCallback(
+    async (element: HTMLElement) => {
+      if (!element || !trackingIsEnabled || eventSent) return;
+      if (!observer.current) await initObserver();
+      (observer.current as unknown as IntersectionObserver)?.observe(element);
+    },
+    [trackingIsEnabled, eventSent],
+  );
 
-    (observer.current as unknown as IntersectionObserver)?.observe(element);
-  };
+  return viewTracker;
 };
 
 export default (eventTrackingData?: EventTrackingData): any => {
   const { isLite } = useContext(RequestContext);
   const viewTracker = getComponentViewTracker(eventTrackingData);
-  const liteATIUrl = constructLiteSiteATIEventTrackUrl({
+  const liteATIUrl = constructStaticATIUrl({
```

Comment:
Shall we rename this field to remove `lite` reference?
```suggestion
  const atiUrl = constructStaticATIUrl({
```
```suggestion
  const staticATIUrl = constructStaticATIUrl({
```